Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_19_0.md

```diff
@@ -0,0 +1,21 @@
+
+#### Scripts
+
+##### New: MicrosoftAtpIsolateMachine
+
+New: Added a new ***MicrosoftAtpIsolateMachine*** script that isolates a machine from accessing external networks.
```

Comment:
Added the **MicrosoftAtpIsolateMachine** script that isolates a machine from accessing external networks.